frontend/components/draft/lesson/lesson-renderer.tsx

```
@@ -179,6 +180,108 @@ export function LessonRenderer({ lesson, dropletSlug }: LessonRendererProps) {
+          break;
+        case "Callout Block":
+          // Map callout type names to colors
+          const calloutColorMap: Record<string, string> = {
+            Warning: "bg-red-300",
+            Question: "bg-blue-300",
+            Important: "bg-orange-300",
+            Definition: "bg-green-300",
+            Information: "bg-purple-300",
+            Caution: "bg-amber-300",
+            Default: "bg-sky-50 dark:bg-sky-200",
+          };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)